cpp/src/utilities/cudf_utils.h

```diff
@@ -36,9 +36,50 @@ bool gdf_is_valid(const gdf_valid_type *valid, gdf_index_type pos) {
 		return true;
 }
 
+/**
+  * Calculates the number of chars used for a validity indicator pseudo-column for a given column's size.
+  *
+  * @node This function is different gdf_get_num_bytes_for_valids_allocation because it refers to bytes used as opposed to allocated
+  *
+  * @param[in] column_size the number of elements
+  * @return the number of bytes necessary to make available for the validity indicator pseudo-column
+  */
 CUDA_HOST_DEVICE_CALLABLE
-gdf_size_type gdf_get_num_chars_bitmask(gdf_size_type size) { 
-	return (( size + ( GDF_VALID_BITSIZE - 1)) / GDF_VALID_BITSIZE ); 
+gdf_size_type gdf_get_num_chars_bitmask(gdf_size_type column_size) { 
```

Comment:
There is already a PR that will work on starting to rework all the bitmask functions, I am afraid of the conflicts that it will create to change the dozens of instances of usage of gdf_get_num_chars_bitmask. That should be left to its own effort.